core/eth/geth_copied.go

```diff
@@ -47,6 +47,12 @@ func parseTopics(out interface{}, fields abi.Arguments, topics []common.Hash) er
 		if !arg.Indexed {
 			return errors.New("non-indexed field in topic reconstruction")
 		}
+
+		_, exists := reflect.TypeOf(out).Elem().FieldByName(capitalise(arg.Name))
```

Comment:
I think we may need a comment on this.

Assuming it happens when the struct is out of sync with the log? When might that happen in real use?

Reply:
Yeah, it's when the struct fields don't match what Geth's decoding logic expects.  This'll happen if we don't mind how we name fields when adding or updating them.